pkg/controller/installation/core_controller.go

```diff
@@ -709,7 +709,7 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		}
 
 		authentication, err = utils.GetAuthentication(ctx, r.client)
-		if err != nil {
+		if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Hm, interesting. For my own understanding, why was it OK before to return an error when not found and not any more?